drivers/src/drv8825.cpp

```diff
@@ -74,4 +66,4 @@ void drv8825::driver_position(hal::degrees p_position){
     long partial_step_difference = p_position/m_partial_steps_to_deg - m_partial_steps;
```

Comment:
This makes me think that m_partial_steps_to_deg may not need to be a float. But maybe I'm wrong. Resolve this without change if this must be a float.